### PR TITLE
Support the option "chunk_size" for "du.Upload".

### PR DESCRIPTION
**Pull request by cainmagi**
1. Add an option `chunk_size` to `du.Upload`. It just simply exposes the `chunkSize` option of the react component.
2. Extend the documentation of `du.Upload` and the changelog.
3. Fix some typos in the documentation.

-----
To author:
1. I have not updated the documentation. If you think this PR is acceptable, I could add a commit for the docs before you merge it.
2. Please blacklify `upload.py` from your side. Thank you!

### CODE REVIEW

**Comment by np-8** [comment]
Thanks for the PR @cainmagi ! I think this is perfectly fine! Yeah, it would perhaps good to include the parameter in the docs before merging (and add comment to changelog). As for the blackification, I'm just going to make a separate commit after the PR merge, and it's fine in this small sized PRs. On larger PRs it would be good to have the same style applied before accepting the PR. 

By the way, what code editor you use? At least on Visual Studio Code there is support for automatic formatting (with black or others) on file save, which can be also configured just for one Workspace (handy!).

**Comment by cainmagi** [comment]
OK, I will implement the docs for this PR first.

I am using VS codes. But I have not tried automatic formatting before. I always use `flake8` and `pylint` to check my codes and fix the warnings manually. In the last week, I have learned how to use `black` from you.

**Comment by cainmagi** [comment]
The documentation and changelog have been added.

File: docs/dash-uploader.md [hunk 1]
```diff
@@ -215,10 +220,9 @@ def call_me(filenames):
 ```
 
 #### output: dash.dependencies.Output
-
 This is the Output object, just as in regular [dash callbacks](https://dash.plotly.com/basic-callbacks).
 
-#### output: str
+#### id: str
```
Comment (np-8) [inline docs/dash-uploader.md:225]: Oh you found out also typo in the docs. Thanks for fixing! :)

File: docs/dash-uploader.md [hunk 2]
```diff
@@ -138,8 +138,12 @@ By default, all filetypes are accepted.
 
 #### max_file_size: numeric
 The maximum file size in Megabytes. Default: 1024 (=1Gb).
-#### default_style: None or dict
 
+#### chunk_size: numeric
+*New in version **0.6.0***
+The chunk size in Megabytes. Optional. Default: 1 (=1Mb).
```
Comment (np-8) [inline docs/dash-uploader.md:144]: Good default as it's also default for the resumable.js component

File: docs/CHANGELOG.md [hunk 3]
```diff
@@ -1,10 +1,14 @@
 # Changelog
 
+## v.0.6.0 (2021-05-04)
```
Comment (np-8) [inline docs/CHANGELOG.md:3]: Could be put also just "WIP" or "0.6.0 (TBD)", but this is OK for now. Just have to remember to update this when the version will actually be released.

**Comment by np-8** [comment]
Thanks again @cainmagi , this is now part of the `dev` branch and will be part of next release of dash-uploader!

**Comment by cainmagi** [comment]
Thank you! I have just changed `0.6.0` to `TBD` in the CHANGELOG. There should be another `0.6.0` in the docs that need to be changed (if you want to change it).

**Comment by np-8** [comment]
Oh yeah I see it in the `chunk_size` parameter description. I guess the next release will be `0.6.0` as there are new feature(s) coming, so most probably the version number will be correct. Have to update it later if not.

**Pull request merged**